site/mosaic.config.js

```diff
@@ -23,6 +23,26 @@ const saltConfig = {
       modulePath: "@jpmorganchase/mosaic-plugins/BreadcrumbsPlugin",
       disabled: true,
     },
+    {
+      modulePath: require.resolve(
+        "./src/mosaic-plugins/LabsComponentPlugin.mjs"
+      ),
+      options: {
+        componentsList: [
+          "Badge",
+          "Combo Box",
+          "Dialog",
+          "Drawer",
+          "Dropdown",
+          "List",
+          "Pill",
+          "Stepped Tracker",
+          "Switch",
+        ],
+        statusLabel: "Lab component",
```

Comment:
I can't see the status label in the preview

Reply:
Where is it meant to appear?

Reply:
Next to the title